examples/saas-clients/amazons3/working-with-amazons3-client/README.md

```diff
@@ -0,0 +1,422 @@
+# Working with Amazon S3 client
+
+## Overview
+The Ballerina Amazon S3 client provides connectivity to Amazon Simple Storage Service API. It allows users to interact 
+with Amazon S3 API to create, store and download and use data with other services. 
+
+This example explains how to use the S3 client to connect with Amazon S3 instance and perform the following operations:
+* Create Bucket
+* Create Object
+* Get Object
+* Delete Object
+* Delete Bucket
+
+## Prerequisites
+- [Ballerina Distribution](https://ballerina.io/learn/getting-started/)
```

Comment:
Any reason why we are not pointing directly to https://ballerina.io/downloads/?

Reply:
Isn't it better go provide step by step guidelines page for the user to download and install ballerina rather than directly providing with the downloads page?